src/api/cart/removeFromCart.js

```diff
@@ -1,19 +1,20 @@
 import axios from "axios";
+import { toastMessages } from "../../utils/toastMessages";
 
-export const removeFromCart = async (cartId, productid, dispatch) => {
+export const removeFromCart = async (cartId, productid, dispatch, token) => {
   try {
-    const {
-      status,
-      // data: {
-      //   cartItems: { _id: cartid, products },
-      // },
-    } = await axios.delete(
-      `https://rabonaserver.joyan11.repl.co/cart/${cartId}/${productid}`
+    dispatch({ type: "CART_ACTION_LOADER", payload: productid });
+    const { status } = await axios.delete(
+      `https://rabonaserver.joyan11.repl.co/cart/${cartId}/${productid}`,
+      { headers: { authorization: token } }
```

Comment:
Try to put the common authorization header for all in a single function or  you can use axios interceptors. 
To know more about this check [here](https://axios-http.com/docs/config_defaults)